mw-webapp/src/logic/themeSwitcher/ThemeSwitche.module.scss

```diff
@@ -0,0 +1,15 @@
+@import "src/styles/variables";
+
+$iconContainerSize: 20px;
+
+.iconWrapper {
+  display: flex;
+  width: $iconContainerSize;
+  height: $iconContainerSize;
+  align-items: center;
+  justify-content: center;
+  border: none;
+  margin: 0 $marginSmall;
```

Comment:
Are you  syre that you need margin left and right for icon?

Reply:
I don't know but changing marging for icon is strange little bit. Inside container with icon, link and buttons should be the same gap, am I wrong?

Reply:
Yes you are right, remove margin